internal/repository/validator.go

```diff
@@ -130,60 +131,89 @@ func (pg *Database) getAllOutputsFromProcessedInputs(
 	return nil, nil
 }
 
-func (pg *Database) FinishEpoch(
+func (pg *Database) SetEpochClaimAndInsertProofsTransaction(
```

Comment:
Is this going to be changed again in @torives' PR?

Reply:
This is already aligned with @torives . This is the actual name.

Reply:
> Is this going to be changed again in @torives' PR?

In an upcoming PR tackling #549, right @fmoura?